wiremock/src/main/java/com/cognifide/aem/stubs/wiremock/WireMockStubs.java

```diff
@@ -194,5 +202,22 @@ public Config getConfig() {
       description = "Set maximum number of entries in request journal (if enabled). When this limit is reached oldest entries will be discarded. 0 means no limits."
     )
     int requestJournalMaxSize() default 200;
+
+    @AttributeDefinition(
+      name = "CORS Enabled",
+      description = "Enable automatic sending of CORS headers")
+    boolean corsEnabled() default true;
+
+    @AttributeDefinition(name = "Allow-Headers",
+      description = "Access-Control-Allow-Headers")
+    String allowHeaders() default "*";
```

Comment:
maybe `AC Allow Headers` etc and prop name `acAllowHeaders`

Reply:
I would keep description n as it is. Like a full header name